src/functionalTest/resources/application-integration.yml

```diff
@@ -148,3 +148,17 @@ launchdarkly:
     key: ${RD_LD_SDK_KEY:}
     environment: ${ENV_NAME:local}
     user: rd
+
+elastic:
+  search:
+    query: ${SEARCH_QUERY_FROM:(roles:judiciary) AND lastModified:>now-72h}
+    recordsPerPage: 500
+
+idam:
+  api:
+    url: ${IDAM_URL:https://idam-api.aat.platform.hmcts.net}
+    client-id: ${OAUTH2_CLIENT_ID:rd-professional-api}
```

Comment:
I think this should not be professional-api

Reply:
for now let it be as JRD-system-user not yet activated. hence using rd-professionla-api now.and it is fine for now